tests/test_guid_based_recommender.py

```diff
@@ -24,14 +28,47 @@ def test_ctx():
     """
     This sets up a basic context for use for testing
     """
-    ctx = Context()
-    ctx['clock'] = Clock()
-    ctx['s3data'] = s3data
-    ctx['cache'] = JSONCache(ctx)
-    return ctx
+    return default_context()
 
 
+@mock_s3
 def test_recommender(test_ctx):
+    SAMPLE_DATA = {
```

Comment:
Can we please replace the test guids with fake guids in the form "guid-01", "guid-02" so we are not leaking information from a real client query.

Reply:
Fixed in new commit.